.github/workflows/android.yml

```diff
@@ -0,0 +1,60 @@
+name: SecureStorage Github Actions CI
+
+on:
+  push:
+    branches-ignore:
+      - 'gh-pages'
+
+jobs:
+  assemble:
+    name: AssembleDebug SecureStorage
+    runs-on: macOS-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+      - name: AssembleDebug SecureStorage
+        run: bash ./gradlew assembleDebug --stacktrace
+
+  test:
+    needs: assemble
+    name: Test SecureStorage
+    runs-on: macOS-latest
+    strategy:
+      matrix:
+        api-level: [21, 22, 23, 25, 29]
+        target: [default]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+      - name: Test SecureStorage
+        uses: reactivecircus/android-emulator-runner@v2
+        with:
+          api-level: ${{ matrix.api-level }}
+          target: ${{ matrix.target }}
+          arch: x86_64
+          profile: Nexus 6
+          script: ./gradlew app:connectedAndroidTest -Pandroid.testInstrumentationRunnerArguments.class=de.adorsys.android.securestoragetest.SecureStorageLogicTest --stacktrace
```

Comment:
build a gradlew tasks

Reply:
?

Reply:
@IsabellaCaspari This will change with the release of SecureStorage2 so I will leave it like this for the meantime

Reply:
Relic from travis CI. Updated it to only `./gradlew connectedAndroidTest --stacktrace`